src/cli/commands/validator.ts

```diff
@@ -11,16 +11,18 @@ export async function handleValidatorCommand(args: ValidatorCommandArgs) {
   let config, configPath
   try {
     ;({ config, configPath } = await resolveConfig(args))
-    if (configPath != null) {
+    if (typeof configPath === 'string') {
       logInfo('Loading config from %s', configPath)
     }
-    if (config.validator == null) {
-      console.error(`This config ${config} is missing a 'validator' property`)
+    if (typeof config.validator == 'undefined') {
```

Comment:
probably makes sense to use strict comparison

Reply:
The original `if (config.validator == null)` checks for both `undefined` and `null` (hence the only reason one should use `==` vs `===` in JS :) ).
So I'd like to keep that.

Reply:
I'd say it will confuse people who are not Javascript Ninja. We write code for people :)

Reply:
If it was by design then I'd use `if (!config.validIator)`

Reply:
This has nothing todo with JS ninja, it is the most explicit way to check if a property is defined. Here I'm checking that it is and that it is set non-null and not `undefined`.
`!` doesn't communicate this clearly and would fail if you deal when checking for the _existence_ of boolean properties.

Reply:
@kurpav We're gonna turn you into a JS ninja 🥷 The way it is written is very common in JS land. If there's a bug here then please demonstrate the bug along with the proposed fix, otherwise this becomes personal preference which we should avoid.

@thlorenz Do we need better prettier / eslint auto fixing stuff in here so we avoid debating code styles flowing forward?